science/nektarpp/Portfile

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               mpi 1.0
+PortGroup               gitlab 1.0
+
+name                    nektarpp
```

Comment:
The `name` line needs to be moved to just below the `gitlab.setup` or else it will get overridden.

Reply:
@mf2k No problem, have moved.